RFC/src/RFC-0201_TariScript.md

```diff
@@ -437,9 +548,9 @@ In addition to the changes given above, there are consensus rule changes for tra
 
 Verify that for every valid transaction or block:
 
-1. The [sender metadata signature] \\(s\_{Mi}\\) is valid for every output.
+1. The [metadata signature] \\( (a_{Ma}, b_{Ma}, R_{Ma}) \\) is valid for every output.
```

Comment:
I would rather stick with `s\_{Mi}` as the metadata signature then starting to point to wards its sub parts. Its more consistent with the rest of the document and easier to read in my mind. We should already have explained that `s\_{Mi} = (a_{Ma}, b_{Ma}, R_{Ma})`. 

For example just below this in "4" we talk of just `\\( s\_{Si} \\),` Everywhere else in the document we just talk of the signature as a single variable.